catalog.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Binabox</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet"/>
+    <link href="https://fonts.googleapis.com/css2?family=Bakbak+One:wght@400;500;600&display=swap" rel="stylesheet"/>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo">
+            <img src="images/logo-white.svg" width="188" height="42" class="logo__image" alt="Binabox">
+        </a>
+        <nav class="header-nav">
+            <ul class="header-nav__wrapper">
+                <li class="header-nav__item">
+                    <a href="#">Home</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Blog</a>
+                </li>
+                <li class="header-nav__item header-nav__item--active">
+                    <a href="#">Catalog</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Contact Us</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="header-actions">
+            <form class="search-form" action="/search">
+                <button type="submit" class="search-form__btn">
+                    <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M21 21L16.65 16.65M19 11C19 15.4183 15.4183 19 11 19C6.58172 19 3 15.4183 3 11C3 6.58172 6.58172 3 11 3C15.4183 3 19 6.58172 19 11Z" stroke="#F5F8FB" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                    </svg>
+                </button>
```

Comment:
Please make search button bigger, because 24px it's too small area for touch devices.
We can easily set 48px X 48px or 44px X 44px.

<img width="828" alt="Screenshot 2024-12-10 at 21 11 31" src="https://github.com/user-attachments/assets/4e9b0011-b817-4732-bdf9-a13f7a00b391">

Reply:
fix it. done
<img width="337" alt="Снимок экрана 2024-12-11 в 22 27 31" src="https://github.com/user-attachments/assets/f10d7cf4-2952-47b3-a8b4-528cc39c367e">